examples/ecs/aws-prometheus/ecs-ec2-adot-config.yaml

```diff
@@ -40,14 +39,16 @@ extensions:
     endpoint: :1888
   zpages:
     endpoint: :55679
+  sigv4auth:
+    region: {{region}}
```

Comment:
Do we need to include the `aps` service field here?

Reply:
The service field is not required as it can inferred from endpoint url